generated/niscope/niscope/_grpc_stub_interpreter.py

```diff
@@ -0,0 +1,425 @@
+        return response.num_wfms
+
+    def add_waveform_processing(self, channel_list, meas_function):  # noqa: N802
+        self._invoke(
+            self._client.AddWaveformProcessing,
+            grpc_types.AddWaveformProcessingRequest(vi=self._vi, channel_list=channel_list, meas_function_raw=meas_function.value),
+        )
+
+    def auto_setup(self):  # noqa: N802
+        self._invoke(
+            self._client.AutoSetup,
+            grpc_types.AutoSetupRequest(vi=self._vi),
+        )
+
+    def cal_fetch_date(self, which_one):  # noqa: N802
+        raise NotImplementedError('cal_fetch_date is not supported over gRPC')
+
+    def cal_fetch_temperature(self, which_one):  # noqa: N802
+        raise NotImplementedError('cal_fetch_temperature is not supported over gRPC')
+
+    def self_cal(self, channel_list, option):  # noqa: N802
+        self._invoke(
+            self._client.CalSelfCalibrate,
+            grpc_types.CalSelfCalibrateRequest(vi=self._vi, channel_list=channel_list, option_raw=option.value),
+        )
+
+    def clear_waveform_measurement_stats(self, channel_list, clearable_measurement_function):  # noqa: N802
+        self._invoke(
+            self._client.ClearWaveformMeasurementStats,
+            grpc_types.ClearWaveformMeasurementStatsRequest(vi=self._vi, channel_list=channel_list, clearable_measurement_function_raw=clearable_measurement_function.value),
+        )
+
+    def clear_waveform_processing(self, channel_list):  # noqa: N802
+        self._invoke(
+            self._client.ClearWaveformProcessing,
+            grpc_types.ClearWaveformProcessingRequest(vi=self._vi, channel_list=channel_list),
+        )
+
+    def commit(self):  # noqa: N802
+        self._invoke(
+            self._client.Commit,
+            grpc_types.CommitRequest(vi=self._vi),
+        )
+
+    def configure_chan_characteristics(self, channel_list, input_impedance, max_input_frequency):  # noqa: N802
+        self._invoke(
+            self._client.ConfigureChanCharacteristics,
+            grpc_types.ConfigureChanCharacteristicsRequest(vi=self._vi, channel_list=channel_list, input_impedance=input_impedance, max_input_frequency=max_input_frequency),
+        )
+
+    def configure_equalization_filter_coefficients(self, channel_list, coefficients):  # noqa: N802
+        self._invoke(
+            self._client.ConfigureEqualizationFilterCoefficients,
+            grpc_types.ConfigureEqualizationFilterCoefficientsRequest(vi=self._vi, channel_list=channel_list, coefficients=coefficients),
+        )
+
+    def configure_horizontal_timing(self, min_sample_rate, min_num_pts, ref_position, num_records, enforce_realtime):  # noqa: N802
+        self._invoke(
+            self._client.ConfigureHorizontalTiming,
+            grpc_types.ConfigureHorizontalTimingRequest(vi=self._vi, min_sample_rate=min_sample_rate, min_num_pts=min_num_pts, ref_position=ref_position, num_records=num_records, enforce_realtime=enforce_realtime),
+        )
+
```

Comment:
Any idea why this one isn't supported? I see we only include it in the Python API as a private method to meet the IVI Spec.

Reply:
It was set as 'codegen_method':'no' because for nimi-python it was 'codegen_method':'private'.

I don't know why the IVI spec would cause it to be private, so that just brings up more questions for me. Should the same restriction not apply to grpc-device? Should we add it to grpc-device, but somehow make it obvious that it's intended to be "private", like by adding a leading underscore?

Reply:
My interpretation is that we didn't actually want to include it as part of our public API, but felt obligated to at least include it as a private method in order to meet the IVI spec. @marcoskirsch will likely know for sure.

Reply:
Yes, it seems to part of our obsolete header, so it makes sense that we didn't want to include it in the public API.

Reply:
I'm fine with not including gRPC support for this, for now. We can always add support later.